source/change-streams/change-streams.rst

```diff
@@ -425,7 +420,10 @@ Once a ``ChangeStream`` has encountered a resumable error, it MUST attempt to re
 - If the ``ChangeStream`` has not received any changes, and ``resumeAfter`` is not specified, and the max wire version is >= ``7``:
 
     - The driver MUST execute the known aggregation command.
-    - The driver MUST specify the ``startAtOperationTime`` key set to the original timestamp from when the changestream was first created.
+    - If the original aggregation command did not include a user-provided ``startAtOperationTime``:
+
+        - The driver MUST specify the ``startAtOperationTime`` key set to the ``operationTime`` saved from the response to the original ``aggregate`` command.
+
```

Comment:
I would collapse this to a single bullet point: "The driver MUST specify the ``startAtOperationTime`` key set to the ``startAtOperationTime`` provided by the user or saved from the original aggregation."